server.js

```diff
@@ -162,6 +162,26 @@ app.get('/api/v1/weapons/:id', async (request, response) => {
   }
 });
 
+app.post('/api/v1/characters', async (request, response) => {
+  const character = request.body;
+
+  for (let requiredParameter of ['name', 'race_id', 'class_id', 'weapon_id']) {
+    if (!character[requiredParameter]) {
+      return response
+        .status(422)
+        .send({ error: `Expected format: { name: <String>, race_id: <Number>, class_id: <Number>, weapon_id: <Number> }. You're missing a "${requiredParameter}" property.` });
+    }
+  }
+
+  response.status(201).json({ character })
+
+  try {
+    const id = await database('characters').insert(character, 'id');
+    response.status(201).json({ id })
```

Comment:
3. Add new character to `characters` table in DB if it's successful new character object is returned in response with status code `200`